firebat/README.md

```
@@ -0,0 +1,10 @@
+# Firebat GUI
+
+![gui default](resources/gui1.png)
+
+![gui decompile](resources/gui2.png)
+
+### GUI known issues
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)